app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -61,149 +62,217 @@ fun CreateActivityScreen(
     listActivityViewModel: ListActivitiesViewModel,
     navigationActions: NavigationActions,
 ) {
-    val IMAGE_PICK_CODE = 1000
-    val CAMERA_CAPTURE_CODE = 1001
-    var title by remember { mutableStateOf("") }
-    var description by remember { mutableStateOf("") }
-    val creator = FirebaseAuth.getInstance().currentUser?.uid ?: ""
-    var location by remember { mutableStateOf("") }
-    var price by remember { mutableStateOf("") }
-    var placesLeft by remember { mutableStateOf("") }
-    var dueDate by remember { mutableStateOf("") }
-    var carouselItems by remember { mutableStateOf(items) }
-    var showDialog by remember { mutableStateOf(false) }
+  val IMAGE_PICK_CODE = 1000
+  val CAMERA_CAPTURE_CODE = 1001
+  var title by remember { mutableStateOf("") }
+  var description by remember { mutableStateOf("") }
+  val creator = FirebaseAuth.getInstance().currentUser?.uid ?: ""
+  var location by remember { mutableStateOf("") }
+  var price by remember { mutableStateOf("") }
+  var placesLeft by remember { mutableStateOf("") }
+  var dueDate by remember { mutableStateOf("") }
+  val carouselItems by remember { mutableStateOf(items) }
+  var showDialog by remember { mutableStateOf(false) }
+  val attendees_: List<SimpleUser> = listOf<SimpleUser>()
+  var attendees: List<SimpleUser> by remember { mutableStateOf(attendees_) }
 
-
-    Scaffold(
+  Scaffold(
       modifier = Modifier.fillMaxSize(),
       topBar = {
         TopAppBar(
             title = { Text("Create a new activity") },
-            navigationIcon = {
-              IconButton(onClick = { navigationActions.goBack() }) {
-                Icon(
-                    imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
-                    contentDescription = "Back")
-              }
-            })
+        )
+      },
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = Route.ADD_ACTIVITY)
```

Comment:
good that you added the bottom bar. I guess it was also added in another PR, but since it is the same, we should not have merge issues.